CMakeLists.txt

```diff
@@ -26,6 +26,21 @@ set (CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/scripts")
 include(haiUtil)
 include(FindPkgConfig)
 
+# Platform shortcuts
+set_if(DARWIN      ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+set_if(LINUX       ${CMAKE_SYSTEM_NAME} MATCHES "Linux")
+set_if(MICROSOFT   WIN32 AND (NOT MINGW AND NOT CYGWIN))
+set_if(SYMLINKABLE LINUX OR DARWIN OR CYGWIN)
+
+# Not sure what to do in case of compiling by MSVC.
+# This will make installdir in C:\Program Files\SRT then
+# inside "bin" and "lib64" directories. At least this maintains
+# the current status. Shall this be not desired, override values
+# of CMAKE_INSTALL_BINDIR, CMAKE_INSTALL_LIBDIR and CMAKE_INSTALL_INCLUDEDIR.
+if (NOT DEFINED CMAKE_INSTALL_LIBDIR)
```

Comment:
You may want to just disable it on Windows and enable everywhere else? You want to also replace include/ etc

Reply:
I was thinking about it, but actually I have never seen any "installing standards" on Windows, especially as it comes to libraries, and GNUInstallDirs works on Windows as well. AFAIK cmake on Windows allows it to install in `C:\Program Files` or kinda like that, followed by the project name and then inside this directory the GNU directory layout is as good as any other. I've moved these platform dependent variables just for that purpose, but then I finally came to the conclusion as described in the above comment.